taipy/common/config/ScenarioConfig.py

```diff
@@ -0,0 +1,46 @@
+import matplotlib.pyplot as plt
+import networkx as nx
+
+
+class ScenarioConfig:
```

Comment:
We already have a Scenario Config class that already contains a DAG. It must be re-used.
The purpose is to add a `draw` method to this class.